sys/targets/targets.go

```diff
@@ -1082,10 +1082,14 @@ var (
 
 const (
 	sourceDirVar = "${SOURCEDIR}"
-	simpleProg   = `
+	simpleCProg  = `
 #include <stdio.h>
 #include <dirent.h> // ensures that system headers are installed
-#include <algorithm> // ensures that C++ headers are installed
 int main() { printf("Hello, World!\n"); }
+`
+	simpleCxxProg = `
+#include <algorithm> // ensures that C++ headers are installed
+#include <vector>
+int main() { std::vector<int> v(10); }
```

Comment:
It won't warn about an unused variable, right?

Reply:
CI would fail if it does.